src/ui-kit/form-controls/date/date.component.ts

```diff
@@ -138,7 +139,7 @@ export class SamDateComponent
       return undefined;
     };
   }
-  
+
   static dateValidation() {
     // Does this code even run?!
```

Comment:
I know this is not new but seeing that comment is a little scary

Reply:
@thomastighe And its public static, so we'll have to track down whether its being used in sam-front-end

Reply:
@thomastighe Actually...I see what its doing now, its a custom validator function that would be used externally. We should probably revisit that design at some point though.